### PR TITLE
Add Infrastructure Helm Chart

### PR DESCRIPTION
**Pull request by shaharblanksela**
Add a new chart 'infra-chart' which includes shared resources for all microservices:
PVC for efs.

Bug: Argocd can't recognize nested charts inside child-chart in umbrella-chart

helm/charts/amplication
├── charts
│   ├── amplication-client
│   ├── amplication-git-pull-request-service
│   ├── amplication-git-pull-service
│   ├── amplication-git-push-webhook-service
│   ├── amplication-server
│   ├── amplication-storage-gateway
│   ├── base
│   ├── infra 
in infra:   │   ├── cp-helm-charts (kafka & zookeeper charts)    ❌
                │   ├── postgresql chart    ❌
                │   ├── pvc.yaml ✅

After searching online and discussing it with a colleague, I couldn't find a valid solution for this.
The previous approach: Kafka chart & Postgres are located in the umbrella chart and the variables to override are in values-common.yaml file. 

I would like to get another opinion on the subject.

### CODE REVIEW

**Review by arielweinberger — APPROVED** [review]
Looks great! I'd wait for a review from @mshidlov since I am lacking some context.

File: helm/charts/services/amplication-server/values.yaml
Comment (mshidlov) [inline helm/charts/services/amplication-server/values.yaml:43]: This should be taken from the infra chart

**Review by mshidlov — CHANGES_REQUESTED** [review]

File: helm/charts/amplication/values-amit.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-amit.yaml:4]: Yes, they are. PV is a resource outside the namespace which connects the PVC request from volume data to be stored in efs.  volume is the name of the folder located in the pod.

File: helm/charts/amplication/values-common.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-common.yaml:85]: Changed. I pulled the folder of the helm chart from my previous branch and this is a value from it.  not relevant changed it to '0'.

File: helm/charts/amplication/values-common.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-common.yaml:183]: Changed. I pulled the folder of the helm chart from my previous branch and this is a value from it. not relevant changed it to '0'.

File: helm/charts/amplication/values-mor.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-mor.yaml:58]: I saw that values-mor is not updated with the rest of the custom-values (like Shimi & Amit) if it doesn't relevant for now I'll restore the values-mor from 'master'.

File: helm/charts/amplication/values-mor.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-mor.yaml:21]: Done

File: helm/charts/amplication/values-prod.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-prod.yaml:49]: restored.

File: helm/charts/amplication/values-staging-os.yaml
Comment (shaharblanksela) [reply helm/charts/amplication/values-staging-os.yaml:118]: It's already in values-common and it doesn't change through environments.

File: helm/charts/services/amplication-git-pull-request-service/values.yaml
Comment (shaharblanksela) [reply helm/charts/services/amplication-git-pull-request-service/values.yaml]: Done

File: helm/charts/services/amplication-git-pull-service/Chart.yaml
Comment (shaharblanksela) [reply helm/charts/services/amplication-git-pull-service/Chart.yaml:11]: No, it's the bug that we can't put nested charts inside another chart as I mentioned above.

File: helm/charts/services/amplication-git-pull-service/values.yaml
Comment (shaharblanksela) [reply helm/charts/services/amplication-git-pull-service/values.yaml:35]: Done

**Pull request merged**
